ADMVEDU/ru.lproj/Localizable.strings

```diff
@@ -0,0 +1,11 @@
+/* 
+  Localizable.strings
+  ADMVEDU
+
+  Created by Bobr, Andrey on 19.05.21.
+  
+*/
+// Errors
+"The provided data is not valid." = "Предоставленные данные не корректные.";
```

Comment:
Let structure keys in format "X.Y.Z" (e.g. "Error.Message.ProvidedDataNotValid")